cmd/soroban-rpc/internal/methods/get_transaction.go

```diff
@@ -41,32 +38,8 @@ type GetTransactionResponse struct {
 	// LatestLedgerCloseTime is the unix timestamp of when the oldest ledger was closed.
 	OldestLedgerCloseTime int64 `json:"oldestLedgerCloseTime,string"`
 
-	// The fields below are only present if Status is not TransactionNotFound.
-
-	// ApplicationOrder is the index of the transaction among all the transactions
-	// for that ledger.
-	ApplicationOrder int32 `json:"applicationOrder,omitempty"`
-	// FeeBump indicates whether the transaction is a feebump transaction
-	FeeBump bool `json:"feeBump,omitempty"`
-	// EnvelopeXDR is the TransactionEnvelope XDR value.
-	EnvelopeXDR  string          `json:"envelopeXdr,omitempty"`
-	EnvelopeJSON json.RawMessage `json:"envelopeJson,omitempty"`
-	// ResultXDR is the TransactionResult XDR value.
-	ResultXDR  string          `json:"resultXdr,omitempty"`
-	ResultJSON json.RawMessage `json:"resultJson,omitempty"`
-	// ResultMetaXDR is the TransactionMeta XDR value.
-	ResultMetaXDR  string          `json:"resultMetaXdr,omitempty"`
-	ResultMetaJSON json.RawMessage `json:"resultMetaJson,omitempty"`
-
-	// Ledger is the sequence of the ledger which included the transaction.
-	Ledger uint32 `json:"ledger,omitempty"`
-	// LedgerCloseTime is the unix timestamp of when the transaction was included in the ledger.
-	LedgerCloseTime int64 `json:"createdAt,string,omitempty"`
-
-	// DiagnosticEventsXDR is present only if Status is equal to TransactionFailed.
-	// DiagnosticEventsXDR is a base64-encoded slice of xdr.DiagnosticEvent
-	DiagnosticEventsXDR  []string          `json:"diagnosticEventsXdr,omitempty"`
-	DiagnosticEventsJSON []json.RawMessage `json:"diagnosticEventsJson,omitempty"`
+	// Many of the fields below are only present if Status is not TransactionNotFound.
```

Comment:
Is this `transactionInfo` with Json?

Reply:
@willemneal This is a Go-ism, the TransactionInfo type is _embedded_ into the GetTransactionResponse, so it's all its fields are flattened into it. There's no `transactionInfo` field.